unit-tests/chunked_iostream_test.cc

```diff
@@ -63,7 +57,7 @@ using namespace libdap;
 
 string the_test_text = "Stephen could remember an evening when he had sat there in the warm,\n"
                        "deepening twilight, watching the sea; it had barely a ruffle on its surface,\n"
-                       "and yet the Sophie picked up enough moving air with her topgallants\n"
+                       "and yet the Sophie picked up enough moving air with her top gallants\n"
```

Comment:
[A topgallant is a sail it's a name. There is no space.](https://www.merriam-webster.com/dictionary/topgallant)
Please don't...

Reply:
Good point. I crewed on a sailing ship - I should have caught that.